### PR TITLE
Update projects.yml

### PR DESCRIPTION
**Pull request by nigelcharman**
Added Concordion Logging Tooltip Extension

### CODE REVIEW

**Comment by shiftkey** [comment]
Thanks!

File: _data/projects.yml [hunk 1]
```diff
@@ -684,3 +684,11 @@
   upforgrabs:
     name: Up for grabs
     link: https://github.com/uhaciogullari/SimpleMvcSitemap/labels/up-for-grabs
+
+- name: Concordion Logging Tooltip Extension
+  desc: An extension to [Concordion](http://concordion.org) that adds tooltips to the output to display logging information.
```
Comment (shiftkey) [inline _data/projects.yml]: I think this is going to break when I merge it. Could you remove the Markdown tags here?

File: _data/projects.yml [hunk 2]
```diff
@@ -684,3 +684,11 @@
   upforgrabs:
     name: Up for grabs
     link: https://github.com/uhaciogullari/SimpleMvcSitemap/labels/up-for-grabs
+
+- name: Concordion Logging Tooltip Extension
+  desc: An extension to Concordion that adds tooltips to the output to display logging information.
+  site: https://github.com/concordion/concordion-logging-tooltip-extension
+  tags: [Java JavaScript HTML CSS BDD SBE testing]
```
Comment (dahlbyk) [inline _data/projects.yml:691]: These elements should be comma-delimited.

**Comment by dahlbyk** [comment]
Another small change. Can you merge in the latest from master too so we can push the big green button?

**Comment by nigelcharman** [comment]
Closing this pull request and replacing with a new one - #159 to make the changes cleaner and easier.

**Pull request closed without merging**